api/api.proto

```diff
@@ -1824,6 +1844,13 @@ service OpenStorageNode {
       };
   }
 
+	// Returns capacity usage of all volumes/snaps for a give node
+	rpc VolumeUsageByNode(SdkVolumeUsageByNodeRequest)
```

Comment:
The messages for the respose and request should be in the following format (I know that some don't do this, but they should :) ):

```
Sdk<Service><Rpc Name><Request/Response>
```

so this would should be `SdkNodeVolumeUsageByNodeRequest`, etc.

if this is not in `docs/dev-sdk.md` please feel free to add it there.